src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -1,67 +1,66 @@
-  --#{$text-input-group}--c-chip-group__main--PaddingBottom: var(--#{$pf-global}--spacer--xs);
+  --#{$text-input-group}--c-chip-group__main--PaddingTop: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}--c-chip-group__main--PaddingRight: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}--c-chip-group__main--PaddingBottom: var(--pf-t--global--spacer--xs);
 
   // Text input
   --#{$text-input-group}__text-input--PaddingTop: var(--#{$pf-global}--spacer--form-element);
-  --#{$text-input-group}__text-input--PaddingRight: var(--#{$pf-global}--spacer--sm);
+  --#{$text-input-group}__text-input--PaddingRight: var(--pf-t--global--spacer--sm);
   --#{$text-input-group}__text-input--PaddingBottom: var(--#{$pf-global}--spacer--form-element);
-  --#{$text-input-group}__text-input--PaddingLeft: var(--#{$pf-global}--spacer--sm);
+  --#{$text-input-group}__text-input--PaddingLeft: var(--pf-t--global--spacer--sm);
   --#{$text-input-group}__text-input--MinWidth: 12ch;
-  --#{$text-input-group}__text-input--m-hint--Color: var(--#{$pf-global}--Color--200); // matches placeholder text color
-  --#{$text-input-group}__text-input--placeholder--Color: var(--#{$pf-global}--Color--200);
+  --#{$text-input-group}__text-input--m-hint--Color: var(--pf-t--global--text--color--default);
+  --#{$text-input-group}__text-input--placeholder--Color: var(--pf-t--global--text--color--placeholder);
   --#{$text-input-group}__text-input--BackgroundColor: transparent;
 
   // Icon
-  --#{$text-input-group}__icon--Left: var(--#{$pf-global}--spacer--sm);
-  --#{$text-input-group}__icon--Color: var(--#{$pf-global}--icon--Color--dark);
+  --#{$text-input-group}__icon--Left: var(--pf-t--global--spacer--sm);
+  --#{$text-input-group}__icon--Color: var(--pf-t--global--icon--color--regular);
   --#{$text-input-group}__icon--TranslateY: -50%;
 
   // Utilities
-  --#{$text-input-group}__utilities--child--MarginLeft: var(--#{$pf-global}--spacer--xs);
-  --#{$text-input-group}__utilities--c-button--PaddingRight: var(--#{$pf-global}--spacer--xs);
-  --#{$text-input-group}__utilities--c-button--PaddingLeft: var(--#{$pf-global}--spacer--xs);
+  --#{$text-input-group}__utilities--child--MarginLeft: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}__utilities--c-button--PaddingRight: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}__utilities--c-button--PaddingLeft: var(--pf-t--global--spacer--xs);
 
   // input disabled style
-  --#{$text-input-group}--m-disabled--Color: var(--#{$pf-global}--disabled-color--100);
-  --#{$text-input-group}--m-disabled--BackgroundColor: var(--#{$pf-global}--disabled-color--300);
+  --#{$text-input-group}--m-disabled--Color: var(--pf-t--global--text--color--on-disabled);
+  --#{$text-input-group}--m-disabled--BackgroundColor: var(--pf-t--global--background--color--disabled--default);
+}
 
+.#{$text-input-group} {
   position: relative;
   display: flex;
+  align-items: flex-start;
```

Comment:
Curious why we need this? Currently it's causing this bit of misalignment:

<img width="864" alt="Screenshot 2024-02-28 at 11 11 34 AM" src="https://github.com/patternfly/patternfly/assets/35148959/8588f091-53cb-4dcc-8cd2-a59d9d1eef81">

<img width="890" alt="Screenshot 2024-02-28 at 11 11 39 AM" src="https://github.com/patternfly/patternfly/assets/35148959/59c04c7c-72f5-4a74-959f-c32e2ed8e65e">

 If it's to keep the close button (or any of the actions on the right side) at the top, it may be better to apply `align-self: flex-start` to the utilities element that wraps those actions.

Reply:
Both the badge and button are wrapped in the same utility class. Adding `flex-start` instead of `center` on the utility class caused the badge to shift to to top as well. I am going to leave it as is for the time being.